lib/cylc/cfgspec/globalcfg.py

```diff
@@ -127,6 +127,17 @@
             'retrieve job logs max size'  : vdr( vtype='string' ),
             'retrieve job logs retry delays': vdr( vtype='interval_minutes_list', default=[] ),
             'task event handler retry delays': vdr( vtype='interval_minutes_list', default=[] ),
+            'local tail command template' : vdr( vtype='string', default="tail -n +1 -F %(filename)s"),
+            'remote tail command template' : vdr( vtype='string', default="tail --pid=`ps h -o ppid $$ | sed -e s/[[:space:]]//g` -n +1 -F %(filename)s"),
+            # Template for tail commands on remote files.
```

Comment:
Note the `sed` expression here - I discovered that the `ps` output has a space preprended on Linux too:
```
bash$ ps h -o ppid $$ 
 2122
```
... surprised this didn't break remote job file tailing at MO too??